packages-web/range-slider/src/components/RangeSliderContainer.ts

```diff
@@ -298,7 +299,7 @@ export default class RangeSliderContainer extends Component<RangeSliderContainer
     }
 
     private isDecimal(attribute: string): boolean {
-        return this.props.mxObject.getAttributeType(attribute) === "Decimal";
+        return this.props.mxObject!.getAttributeType(attribute) === "Decimal";
```

Comment:
Maybe is better to use `this.props.mxObject ?? this.props.mxObject.getAttributeType(attribute) === "Decimal"`

Reply:
In the current implementation, there is no path without an mxObject.... should be all right, let me check if can write it in a different way